pkgs/development/libraries/portaudio/default.nix

```diff
@@ -12,30 +13,26 @@ stdenv.mkDerivation rec {
     ++ stdenv.lib.optional (!stdenv.isDarwin) alsaLib;
 
   configureFlags = stdenv.lib.optionals stdenv.isDarwin
-    [ "--build=x86_64" "--without-oss" "--enable-static" "--enable-shared" ];
+    [ "--disable-mac-universal" ];
```

Comment:
Does this flag do any harm when passed into a non-darwin build? I generally try to minimize platform-conditional stuff.